### ISSUE TITLE
Example with multiple coordinate sets / animations?

### ISSUE THREAD

**Issue by ghutchis** [OpenChemistry/chemicaljson#5]
Are there examples of CJSON with multiple conformers / coordinate sets?

**Comment by ghutchis** [comment]
Bump. There doesn't seem to be any way to support multiple coordinate sets in CJSON, which is a big problem for animations, dynamics, conformers, etc.

**Comment by wadejong** [comment]
One good way to introduce a series of coordinate sets would be to introduce some kind of iterator, including a keyword in the geometry identifying the type of coordinates, xyz, unit cell, and if this is a coordinate series (could be both xyz or unit cell in case of ab initio molecular dynamics). We will need to add a time stamp (from MD) or an iterator integer to ensure we can extract them in the correct time order.

Bert

**Comment by ghutchis** [comment]
Are we going to address this in CJSON or work on the new QC JSON and handle it there?

**Comment by cryos** [comment]
I was going to get this in here, but got distracted by switching to a more modern JSON library. I think what we do here/in QC JSON will hopefully share a lot. Thanks for the reminder, it should not have taken this long, apologies! I think this would offer a simple way of getting smaller trajectories out of other packages in that case too.

**Comment by cryos** [comment]
I have working code now, and will get the examples uploaded soon.

**Comment by ghutchis** [comment]
This seems to already be in the wild as part of `3dSets` .. an array of arrays.

What's missing:
- time step (from MD)
- relative energies or other properties (e.g., recording forces through an optimization or MD)
- a way to indicate the default set (e.g., usually with geometry optimization, the default `3d` set would indicate the best, lowest-energy geometry .. but in MD, the default might duplicate the first coordinates.)
  - In other words, if `3dSets` is available, can something else indicate the default index?

**Comment by Shruthii7** [comment]
Hi! From the discussion, it looks like multiple coordinate sets are already supported via 3dSets, but there isn’t a clear example or documentation showing how this works in practice.

**I want to contribute by adding:**
- a minimal example JSON demonstrating 3dSets (e.g., simple conformers/animation), and
- a short docs section explaining current usage and explicitly noting the existing limitations.

This wouldn’t propose schema changes, just document existing behavior.

**Comment by Shruthii7** [comment]
@ghutchis, I’ve opened PR #33, which adds documentation and a minimal example
for the current 3dSets (array-of-arrays) approach and documents the known limitations discussed here.

Happy to extend this further (like default set index, timestep,
or associated properties) If that would be useful.
